modules/sildisco/www/sp/discoresp.php

```diff
@@ -1,7 +1,8 @@
 <?php
 /**
  * Modified version of modules/saml/www/sp/discoresp.php
- * 2022-09-26 -- merged with SimpleSAMLphp 1.19.6
+ * 2024-06-06 -- merged with SimpleSAMLphp 1.19.8
+ * Changes marked with "GTIS"
```

Comment:
This was slightly different than the others.

```suggestion
 * 2024-06-06 -- Merged with simplesamlphp 1.19.8, lines marked with GTIS are modified
```